static/js/publisher/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from "react";
+
+function useLocalStorage<T = unknown>(
```

Comment:
Nice addition, but is it used anywhere? I can't seem to find the usage in the PR.